Pool and Pool stable: adds check if the swapped token is actualy one of the two tokens in the liquidity pool. (#237)
Phoenix-Protocol-Group/phoenix-contracts: CHANGELOG.md

## Added

- Adds a new macro that validates the bps arguments value ([#199])

> - Pool and Pool stable: adds check if the swapped token is actualy one of the two tokens in the liquidity pool ([#237])

Reviewer: ```suggestion
- Pool and Pool stable: extend check if the swapped token is within pool ([#237])
```
